tests/phpunit/abilities/class-wc-stripe-ability-base-test.php

```
@@ -0,0 +1,241 @@
+		$this->assertSame( '', $result );
+	}
+
+	public function test_build_expand_fragment_emits_stripe_bracket_form() {
+		// Stripe expects `expand[]=foo&expand[]=bar` — brackets, no index.
+		// http_build_query's default indexed form would not be accepted.
+		$result = WC_Stripe_Ability_Base_Test_Fixture::call_build_expand_fragment(
+			[ 'customer', 'balance_transaction' ]
+		);
+
+		$this->assertSame( 'expand[]=customer&expand[]=balance_transaction', $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_build_expand_fragment_emits_stripe_bracket_form is not named in camelCase. (undefined)

(CamelCaseMethodName)